entc/integration/ent/template/meta.tmpl

```diff
@@ -0,0 +1,20 @@
+{{/* A template for adding additional fields to specific types. */}}
+{{ define "meta/additional/enums" }}
+	{{- range $f := $.Fields }}
+		{{- if $f.IsEnum }}
+			{{ $enum := trimPackage $f.Type.String $.Package }}
```

Comment:
```suggestion
			{{ $enum := trimPackage $f.Type.String $.Package -}}
```

Reply:
It's not really needed here, because gofmt will remove the spaces between methods.